src/Imath/ImathExport.h

```diff
@@ -3,15 +3,58 @@
 // Copyright Contributors to the OpenEXR Project.
 //
 
+// See the document in OpenEXR docs/SymbolVisibility.md for background
+// on all these types
 #if defined(IMATH_DLL)
 #    if defined(IMATH_EXPORTS)
 #        define IMATH_EXPORT __declspec(dllexport)
 #        define IMATH_EXPORT_CONST extern __declspec(dllexport)
+//#        if defined(__MINGW32__)
```

Comment:
What's with the commented-out lines? If it's intentional, shouldn't we remove them entirely?

Reply:
I didn't know if I needed them at the time, dead code has been eliminated